projects/igniteui-angular/migrations/migration-collection.json

```diff
@@ -123,8 +123,13 @@
         },
         "migration-25": {
             "version": "14.1.0",
-            "description": "Updates Ignite UI for Angular from v13.2.x to v14.1.0",
+            "description": "Updates Ignite UI for Angular from v14.1.x to v14.2.0",
```

Comment:
Shouldn't this be `v14.0.x to v14.1.x` ?

Reply:
Sorry, happened during merge. Reverted it back to how it was  before  - `13.2.x to 14.1.0`.